config/disabled.yml

```diff
@@ -34,6 +34,10 @@ Layout/MultilineAssignmentLayout:
   StyleGuide: '#indent-conditional-assignment'
   Enabled: false
 
+Lint/InlineAccessModifier:
+  Description: "Checks that access modifiers are inlined for clarity."
```

Comment:
It would be better to unify them with single quotes.

```diff
-"Checks that access modifiers are inlined for clarity."
+'Checks that access modifiers are inlined for clarity.'
```

Reply:
👍 Sorry, I use double-quotes elsewhere—hard to beat that reflex.